source/set-field-type.d.ts

```diff
@@ -0,0 +1,37 @@
+import type {Except} from './except';
+import type {Simplify} from './simplify';
+
+/**
+Create a type that changes the type of the given keys.
+
+Use-cases:
+- Creating variations of a base model.
+- Fixing incorrect external types.
+
```

Comment:
Add a tip about using the `Merge` type if you need to change multiple properties to different types.